indico/modules/events/papers/controllers/display.py

```diff
@@ -53,3 +60,122 @@ def _process(self):
+        list_config = self.list_generator.list_config
+        return WPDisplayJudgingArea.render_template('management/assignment_list_filter.html', self._conf,
+                                                    event=self.event_new,
+                                                    static_items=self.list_generator.static_items,
+                                                    filters=list_config['filters'],
+                                                    visible_items=list_config['items'])
+
+    def _process_POST(self):
+        self.list_generator.store_configuration()
+        return jsonify_data(flash=False, **self.list_generator.render_list())
+
+
+class RHDisplayPapersActionsBase(RHJudgingAreaBase):
+    """Base class for RHs performing actions on selected contributions"""
+
+    def _checkParams(self, params):
+        RHJudgingAreaBase._checkParams(self, params)
+        ids = map(int, request.form.getlist('contribution_id'))
+        self.contributions = Contribution.query.with_parent(self.event_new).filter(Contribution.id.in_(ids)).all()
+
+
+class RHDisplayBulkPaperJudgment(PaperJudgmentMixin, RHDisplayPapersActionsBase):
+    def _checkParams(self, params):
+        RHDisplayPapersActionsBase._checkParams(self, params)
+        self.list_generator = PaperJudgingAreaListGeneratorDisplay(event=self.event_new, user=session.user)
+
+
+class RHDisplayPapersDownload(PapersDownloadAttachmentsMixin, RHDisplayPapersActionsBase):
+    pass
+
+
+class RHRJudgingAreaAssigningBase(RHDisplayPapersActionsBase):
+    """Base class for assigning/unassigning paper reviewing roles"""
+
+    def _checkParams(self, params):
+        RHDisplayPapersActionsBase._checkParams(self, params)
+        self.role = PaperReviewingRole[request.args['role']]
+
+    def _render_template(self, person_list, action):
+        user_competences = self.event_new.cfp.user_competences
+        competences = {'competences_{}'.format(user_id): competences.competences
+                       for user_id, competences in user_competences.iteritems()}
+        return jsonify_template('events/papers/management/assign_role.html', event=self.event_new, role=self.role.name,
+                                action=action, person_list=person_list, competences=competences,
+                                contribs=self.contributions)
+
+
+class RHJudgingAreaAssign(RHRJudgingAreaAssigningBase):
+    """Dialog with person list to assign paper reviewing roles"""
+
+    def _process(self):
+        if self.role == PaperReviewingRole.content_reviewer:
+            person_list = self.event_new.cfp.content_reviewers
+        elif self.role == PaperReviewingRole.layout_reviewer:
+            person_list = self.event_new.cfp.layout_reviewers
+        else:
+            raise BadRequest
+        return self._render_template(person_list, 'assign')
+
+
+class RHJudgingAreaUnassign(RHRJudgingAreaAssigningBase):
+    """Dialog with person list to unassign paper reviewing roles"""
```

Comment:
I don't think that the docstring should be coupled with the UI. Just writing *"Renders the person list to unassign paper reviewing roles"* would be enough IMO